src/Bridges/Objective/slack.jl

```diff
@@ -0,0 +1,76 @@
+"""
+    SlackBridge{T, F, G}
+
+The `SlackBridge` converts an objective function of type `G` into a
+[`SingleVariable`](@ref) objective by creating a slack variable and a
+`F`-in-[`LessThan`](@ref) constraint for minimization or
+`F`-in-[`LessThan`](@ref) constraint for maximization where `F` is
+`MOI.Utilities.promote_operation(-, T, G, MOI.SingleVariable}`.
+Note that when using this bridge, changing the optimization sense
+is not supported. Set the sense to `MOI.FEASIBILITY_SENSE` first
+in order to delete the bridge in order to change the sense.
```

Comment:
Set the sense to `MOI.FEASIBILITY_SENSE` first to delete the bridge in order to change the sense, then re-add the objective.